presto-main/src/test/java/com/facebook/presto/resourcemanager/TestResourceManagerClusterStateProvider.java

```diff
@@ -483,7 +483,7 @@ private NodeStatus createNodeStatus(String nodeId, MemoryPoolId memoryPoolId, Me
                 "environment",
                 false,
                 new Duration(1, SECONDS),
-                "http://exernalAddress",
+                "http://externalAddress",
                 "http://internalAddress",
```

Comment:
I generally flag things like these for not using `.example`, or similar.

https://datatracker.ietf.org/doc/html/rfc2606#section-2